src/Compilers/Core/Portable/Operations/ControlFlowGraph.cs

```diff
@@ -89,56 +89,63 @@ public sealed partial class ControlFlowGraph
         /// </summary>
         public ControlFlowGraph GetLocalFunctionControlFlowGraph(IMethodSymbol localFunction)
         {
-            // PROTOTYPE(dataflow): It looks like the indexing below will throw right exceptions on all invalid inputs.
-            //                      However, we should consider if we want to perform our own validation for the input.
-            //                      In any case, we need to add unit tests to cover behavior of this API with respect to
-            //                      an invalid input (including null).
+            if (localFunction is null)
+            {
+                throw new ArgumentNullException(nameof(localFunction));
```

Comment:
> I don't see tests covering the invalid argument cases?

What about the only test added in this PR, LambdaFlow_04?

---
In reply to: [193931270](https://github.com/dotnet/roslyn/pull/27604#discussion_r193931270) [](ancestors = 193931270)

Reply:
I must have missed the assert nulls. It would be good to break those out into separate tests though, as that test is probably testing too many things at the same time.